app/assets/javascripts/select-kit/addon/components/category-drop.js

```diff
@@ -54,6 +54,37 @@ export default ComboBoxComponent.extend({
     return this.options.subCategory || false;
   }),
 
+  shortcuts: computed(
```

Comment:
Will you be able to expand on why this change is required and what problem does this change solve? :bow:

Reply:
There are a few branches like `if (this.siteSettings.lazy_load_categories) { ... } else { ... }` and this refactoring made me duplicate less code between the two branches.